stingray/stats.py

```diff
@@ -0,0 +1,492 @@
+import warnings
+from collections.abc import Iterable
+
+import numpy as np
+from scipy import stats
+from stingray.utils import simon
+from stingray.utils import vectorize, float64, float32, int32, int64
+
+
+__all__ = ['p_multitrial_from_single_trial',
+           'p_single_trial_from_p_multitrial',
+           'fold_profile_probability',
+           'fold_detection_level',
+           'z2_n_detection_level',
+           'z2_n_probability',
+           'classical_pvalue']
```

Comment:
should `pds_probability` be in this list, too, since it's used in `Powerspectrum`?